WooCommerce/Classes/ViewRelated/Authentication/AccountCreationForm.swift

```diff
@@ -0,0 +1,140 @@
+    @Environment(\.customOpenURL) var customOpenURL
+
+    /// Triggered when the account is created and the app is authenticated.
+    var completion: (() -> Void) = {}
+
+    @ObservedObject private var viewModel: AccountCreationFormViewModel
+
+    @State private var isPerformingTask = false
+
+    init(viewModel: AccountCreationFormViewModel) {
+        self.viewModel = viewModel
+    }
+
+    var body: some View {
+        ScrollView {
+            VStack(alignment: .leading, spacing: Layout.verticalSpacing) {
+                // Header.
+                VStack(alignment: .leading, spacing: Layout.horizontalSpacing) {
+                    Text(Localization.title)
+                        .largeTitleStyle()
+                    Text(Localization.subtitle)
+                        .foregroundColor(Color(.secondaryLabel))
+                        .bodyStyle()
+                }
+
+                // Form fields.
+                VStack(spacing: Layout.verticalSpacingBetweenFields) {
+                    AccountCreationFormFieldView(viewModel: .init(header: Localization.emailFieldTitle,
+                                                                  placeholder: Localization.emailFieldPlaceholder,
+                                                                  keyboardType: .emailAddress,
+                                                                  text: $viewModel.email,
+                                                                  isSecure: false,
+                                                                  errorMessage: viewModel.emailErrorMessage))
+                    AccountCreationFormFieldView(viewModel: .init(header: Localization.passwordFieldTitle,
+                                                                  placeholder: Localization.passwordFieldPlaceholder,
+                                                                  keyboardType: .default,
+                                                                  text: $viewModel.password,
+                                                                  isSecure: true,
+                                                                  errorMessage: viewModel.passwordErrorMessage))
+                    AttributedText(tosAttributedText)
+                        .attributedTextLinkColor(Color(.textLink))
+                }
+
+                // CTA to submit the form.
+                Button(Localization.submitButtonTitle) {
+                    Task { @MainActor in
+                        isPerformingTask = true
+                        let result = await viewModel.createAccount()
+                        isPerformingTask = false
+                        switch result {
+                        case .success:
+                            completion()
+                        case .failure:
+                            break
+                        }
+                    }
+                }
+                .buttonStyle(PrimaryButtonStyle())
```

Comment:
We should use `PrimaryLoadingButtonStyle` to show the loading state when calling network requests.

Reply:
TIL 👍 updated in https://github.com/woocommerce/woocommerce-ios/pull/7906/commits/bedb52ca747c919b9726d118717bf6912343e824